OpenOversight/app/auth/views.py

```diff
@@ -62,7 +71,7 @@ def before_request():
 def unconfirmed():
     if current_user.is_anonymous or current_user.confirmed:
         return redirect(url_for("main.index"))
-    if current_app.config["APPROVE_REGISTRATIONS"]:
+    if BaseConfig.APPROVE_REGISTRATIONS:
```

Comment:
Found a bug.

Reply:
Can you elaborate on what the bug here was?

Reply:
`Bug` is the wrong term to use, thanks for pointing that out. I believe it would be better to use the config object syntax above than the more dict like syntax that existed prior.

Reply:
I do think there should be a different way to refer to this than `BaseConfig` since it doesn't take into consideration objects that inherit it, but we can save that for later on.

Reply:
I would prefer you undo this current_app.config -> BaseConfig change. Generally I agree with your thought of having an object with attributes is better, but this is not how BaseConfig is intended to be used at the moment, so I think this will break some things, and overall it seems like a minor improvement to me that we can put at the end of our backlog